src/ui/src/builder/settings/BuilderTemplateInput.spec.ts

```
@@ -87,10 +92,15 @@ describe("BuilderTemplateInput", () => {
 			]);
 		});
 
-		it("should autocomplete with templating", async () => {
-			const wrapper = mountWrapper();
+		it.only("should autocomplete with templating", async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)